README.md

```diff
@@ -1,2 +1,49 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
 # Portfolio
-My portfolio that shows my educational background, projects and technical skills.
+
+> This is a mobile-first portfolio project which is built to show projects, background and the ways to get in touch with the developer.
+
+## Built With
+
+- Major languages: HTML & CSS
+- Frameworks: N/A
+- Technologies used: Git & VScode
+
+## Live Demo (if available)
+
+N/A
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+- clone this repo to your local machine.
+- open it with a text editor (VScode)
+
+## Authors
+
+- GitHub: [@khawanin](https://github.com/Khawanin)
+- LinkedIn: [@khawaninali](https://www.linkedin.com/in/khawanin-ali-zada-93777b21a/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```

Comment:
- [ ] What do you think if you remove this line in your README as doesn't appear in the REQUIRED **[README template provided by Microverse](https://github.com/microverseinc/readme-template/)**